server/views/pages/externalLink.html

```diff
@@ -18,16 +18,16 @@
     {# The default og:image is added below head so that scrapers see any custom metatags first, and this is just a fallback #}
     <meta property="og:image" content="{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
 
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-152065860-4"></script>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-H1MT63QRLQ"></script>
```

Comment:
Presuming this wants to be changed the same way as `server/views/components/basicTemplate.njk` and passed in. Looks like it should've done that in the first place 🙂